corehq/ex-submodules/casexml/apps/stock/utils.py

```diff
@@ -69,30 +69,13 @@ def get_current_ledger_transactions_multi(case_ids):
     if not case_ids:
         return {}
 
-    relevant_transactions = StockTransaction.objects.raw(
-        """
-        SELECT MAX(stx.id) AS id FROM
-        (
-            SELECT case_id, product_id, section_id, MAX(sr.date) AS date
-            FROM stock_stocktransaction st JOIN stock_stockreport sr ON st.report_id = sr.id
-            WHERE case_id IN %s
-            GROUP BY case_id, section_id, product_id
-        ) AS x INNER JOIN stock_stocktransaction AS stx ON
-            stx.case_id = x.case_id
-            AND stx.product_id = x.product_id
-            AND stx.section_id = x.section_id
-        JOIN stock_stockreport str ON
-            str.date = x.date
-            AND stx.report_id = str.id
-        GROUP BY stx.case_id, stx.product_id, stx.section_id, str.date;
-        """,
-        [tuple(case_ids)]
-    )
+    results = StockTransaction.objects.filter(
+        case_id__in=case_ids
```

Comment:
if we use `tuple(case_ids)` wouldn't that make it so we could drop the `.distinct`? i know sometimes those can be expensive